### ISSUE TITLE
[BUG] Screen off during videos

### ISSUE THREAD

**Issue by harishbewe** [TheHCJ/Quacker#11]
For me, while watching videos, the screen goes off as per my timeout setting. But isn't it that it should not happen like that. 

Just look into that please.

**Comment by TheHCJ** [comment]
That's a bug, I'll have a look into fixing this

**Comment by TheHCJ** [comment]
That's a bug, I'll have a look into fixing this

**Comment by TheHCJ** [comment]
Out of interest are you watching these videos in full screen?

**Comment by harishbewe** [comment]
No, I haven't tried anything in full screen. I found this thing only in normal feed view.

**Comment by TheHCJ** [comment]
Ok, thanks for that infomation

**Comment by TheHCJ** [comment]
@harishbewe could you tell me if this was fixed in [v3.4.0](https://github.com/TheHCJ/Quacker/releases/tag/v3.4.0)

**Comment by harishbewe** [comment]
@TheHCJ Thanks for fixing it.  Now it works fine.
